src/api/forms/form-metadata-repository.js

```diff
@@ -0,0 +1,51 @@
+import * as fs from 'fs/promises'
+import { join } from 'path'
+import { config } from '~/src/config'
+import * as Types from '../types.js'
+
+// TODO look at prisma to generate this
+
+const formDirectory = config.get('formDirectory')
+
+/**
+ * Returns the filename for a form metadata entry, given a form ID.
+ * @param {string} formId ID of the form
+ * @returns {string} file path
+ */
+const getFormMetadataFilename = (formId) => {
+  return join(formDirectory, `${formId}-metadata.json`)
+}
+
+/**
+ * Retrieves a file from the form store
+ * @returns {Promise<Types.FormConfiguration[]>} form configuration
+ */
+const listForms = async () => {
```

Comment:
Would you mind adding the export keyword to each export?

Ideally export functions with names so they appear in stack traces

```suggestion
export async function listForms() {
```